kartothek/serialization/_parquet.py

```diff
@@ -69,8 +69,22 @@ class ParquetSerializer(DataFrameSerializer):
     _PARQUET_VERSION = "2.0"
     type_stable = True
 
-    def __init__(self, compression="SNAPPY", chunk_size=None):
+    def __init__(
+        self, compression: str = "SNAPPY", chunk_size: Optional[int] = None
+    ) -> None:
+        if compression not in ["NONE", "SNAPPY", "GZIP", "BROTLI", "LZ4", "ZSTD"]:
```

Comment:
I would assume this list is exported by arrow. Can you please check and reuse an arrow interface? Which algo is supported is 100% dictated by arrow and depends even on how it was built, therefore, I would prefer a more dynamic approach, if any

Reply:
I didn't find an exported list by arrow on first glance, perhaps because it is indeed based on the arrow build. I will just remove this check